wgs/wg-pipelines/wg-charter-pipelines.md

```diff
@@ -0,0 +1,96 @@
+# ML Pipelines Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+The goal of the Kubeflow Pipelines project is to build a Kubernetes optimized stack for creating and deploying ML Pipelines and enabling a rich ecosystem of pipelines components and tools. ML Pipelines project has the goal of enabling simple and reliable productionalization of autonomous ML workflows, advancing the standards of ML Engineering and making them available to a large community of users across various Kubernetes deployment options.
+
+
+### In scope
+
+#### Code, Binaries and Services
+
+##### Backend Servers
+- ML Pipelines user facing APIs 
+- Storage abstraction and integration components for pipelines
+- Pipelines Orchestration Engine
+Includes Argo and Tekton based forks, as well as new versions of orchestrators
+- Metadata Store that tracks pipeline artifacts, their lineage and dependencies as well as the ML Metadata Service exposing the Metadata APIs, and the Metadata collection agent
+- Pipeline job scheduling controller, and continuous pipeline execution services
+
+##### FE and UI
+- Pipelines Templates UI
+- Experiment tracking and run exploration UI
+- Metadata Tracking and Lineage Exploration UI
+- Metadata metric visualization 
+
+##### Tools
+- Authoring SDK for components and pipelines
+- Tools for pipeline creation and deployment
+- CLI for interacting with Pipelines API
+
+##### Deployment
+- Deployment scripts and tools for deploying ML Pipeline service components on Kubernetes engines of various cloud providers and on-prem deployments
+
+##### Pipeline Content
+- Cloud specific integration components
+- HW and SW vendor specific components
+- Pipeline examples for different ML use cases
+
+##### Documentation
+- Concepts documentation 
+- Cloud provider specific documentation
+- Documentation for the SDK and REST API
+- Documentation for the sample components and pipelines 
+
+
+#### Dependencies
+
+ML Pipeline projects may have dependencies on the following projects managed by other KF work groups:
```

Comment:
Could you explain the dependencies here? Are you talking about reusable component?

Reply:
These mean to represent dependencies that some users of ML Pipeline may need to complete their scenarios. For example, the multi-user support require the profile controller, the pipelines that run distributed training in the cluster depend on the TFJob controller, etc. The dependencies are not hard, so the ML Pipelines can function without them.

Reply:
updated the markdown here to detail the scenario